baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -18,17 +18,17 @@ This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]]
 
 ## Definition
 
-```
+```ABNF
 list        = list-member 0*179( OWS "," OWS list-member )
 list-member = key OWS "=" OWS value *( OWS ";" OWS property )
 property    = key OWS "=" OWS value
 property    = key OWS
-key         = <token, defined in [[RFC2616], Section 2.2](https://tools.ietf.org/html/rfc2616#section-2.2)>
+key         = token ; as defined in RFC 2616, Section 2.2
 value       = %x21 / %x23-2B / %x2D-3A / %x3C-5B / %x5D-7E
               ; US-ASCII characters excluding CTLs,
               ; whitespace, DQUOTE, comma, semicolon,
               ; and backslash
-OWS         = <Optional white space, as defined in [[RFC7230], Section 3.2.3.](https://tools.ietf.org/html/rfc7230#section-3.2.3)>
+OWS         = *( SP / HTAB ) ; optional white space, as defined in RFC 7230, Section 3.2.3
```

Comment:
I would just add the links after this, e.g. (without any formatting)

* RFC2616, Section 2.2 - https://tools.ietf.org/html/rfc2616#section-2.2
* RFC7230, Section 3.2.3 - https://tools.ietf.org/html/rfc7230#section-3.2.3

Reply:
Both links are already embedded in the surrounding text. Should I list them again after the code block?

Reply:
If you mean `Header Content` section above, I would actually remove RFC links from there (except for ABNF RFC) and keep them here, closer to the usage. Also the section mentions DIGIT (which is not used), but not DQUOTE, which is used (couldn't we just say double-quote?).

Reply:
I moved the OWS link to below the table and made sure to reference RFC 2616, which defines token, using the ReSpec syntax for normative references (`[[!RFC2616]]`) to fix #35.

DQUOTE is a core rule of ABNF defined in the respective RFC but I would be fine with _double quote_ as well since it's just used in a comment anyway. I removed the obsolete mention of DIGIT.